draft-irtf-cfrg-ristretto255-decaf448.md

```diff
@@ -117,7 +116,8 @@ overhead, and only in the encoding and decoding phases.
 
 While Ristretto is a general method, and can be used in conjunction
 with any Edwards curve with cofactor 4 or 8, this document specifies
-the ristretto255 group, which MAY be implemented using Curve25519.
+the ristretto255 group, which MAY be implemented using Curve25519,
+and the decaf448 group, which MAY be implemented using edwards448.
```

Comment:
```suggestion
the ristretto255 group, which can be implemented using Curve25519,
and the decaf448 group, which can be implemented using edwards448.
```

Normative MAY doesn't feel right here. This is talking about an implementation strategy and not an implementation allowance, so just saying that Curve25519 or edwards448 can be used seems better.

Reply:
Agreed, much better.